textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexMatcherTest.java

```diff
@@ -149,17 +163,180 @@ public void testGetNextTupleStaffEmail() throws Exception {
 		fields.add(spanField);
 		expectedResults.add(new DataTuple(spanSchema, fields.toArray(new IField[fields.size()])));
 		
-		//expected to match "180.149.134.141"
+		//expected to match "hwangl@ics.uci.edu"
```

Comment:
A general suggestion: since our software is open source, such email addresses will appear in the public domain and may be subject to spam.  It will be good to make them not real, e.g., `hwangl@abc.def.com`.